ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/common-settings.md

```diff
@@ -87,6 +87,11 @@ mac_address_table:
 platform_speed_groups:
   < platform >:
     < speed >: [ < speed_group >, < speed_group > ]
+
+# Set SNMP settings | Optional
+snmp_settings:
+  contact: < contact_info >
+  location: < boolean | default -> false >
```

Comment:
Could we add a description to explain role of location as boolean? From the code I understand it enables logic to build location string.